src/plugins/vis_type_vislib/public/vislib/components/tooltip/_pointseries_tooltip_formatter.js

```diff
@@ -55,7 +56,15 @@ export function pointSeriesTooltipFormatter() {
 
     if (datum.y !== null && datum.y !== undefined) {
       const value = datum.yScale ? datum.yScale * datum.y : datum.y;
-      addDetail(currentSeries.label, currentSeries.yAxisFormatter(value));
+      let label = currentSeries.label;
```

Comment:
seems data.yAxisLabel will give the label directly?
is there a specific reason to use datum?

```
data: 
{xAxisOrderedValues: Array(5), xAxisFormat: {…}, xAxisLabel: 'machine.os.keyword: Descending', yAxisFormat: {…}, yAxisLabel: 'Count', …}
hits: 25
series: (15) [{…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}]
xAxisFormat: {id: 'terms', params: {…}}
xAxisFormatter: val => xConverter.convert(val)
xAxisLabel: "machine.os.keyword: Descending"
xAxisOrderedValues: (5) ['win xp', 'osx', 'win 7', 'win 8', 'ios']
yAxisFormat: {id: 'number'}
yAxisFormatter: val => yConverter.convert(val)
yAxisLabel: "Count"
zAxisFormat: {id: 'terms', params: {…}}
zAxisFormatter: val => zConverter.convert(val)
zAxisLabel: ""
[[Prototype]]: Object
```

Reply:
Unfortunately, when we have multiple fields on the y axis, the `yAxisLabel` is incorrectly set. The only way we have to correctly get the label then is from the `yRaw` value. Thats why i check in the next few lines to check to see if we have more than one y-axis field before using the `yRaw` value instead of the default label. I've added the comment below to call this out as well.